needrestart

```diff
@@ -1261,6 +1282,25 @@ if(defined($opt_l) && !$uid) {
 	    }
 	}
     }
+
+	## GUESTS
+	$ui->vspace();
+	if (! @guests) {
+		$ui->notice(__ 'No VM guests are running outdated binaries.') unless($opt_b || $opt_m eq 'e');
```

Comment:
I wonder about the wording. Might be misunderstood that needrestart scans inside of guests?

Reply:
Hmm, I see.
How about adding "hypervisor" to be more specific.
  "No VM guests are running outdated hypervisor binaries."
Since we explicitly work with qemu we could say so
  "No VM guests are running outdated hypervisor (qemu) binaries."
  or just
  "No VM guests are running outdated qemu binaries."
Or even if you prefer we can add "on this host"
  "No VM guests are running outdated qemu binaries on this host."
(and similar for the other case below)

What do you think?

Reply:
Sorry for the long delay - I would be OK with most of them :-)

Reply:
"most" isn't all :-)
So should I update the PR to use these messages or do you want something else?

Reply:
@liske I have picked the most expressive combination of my suggestions and added it to the PR.
This also made me find some whitespace damage which I fixed as well.
Furthermore I rebased to the latest so that you can merge this more easily.

Please let me know if you'd need/want anything else to go on with this.